ui/layouts/default.vue

```diff
@@ -72,9 +200,24 @@ export default {
     return {
       isBrowser: false,
       isTouchScreen: false,
-      lastTouchTime: 0
+      lastTouchTime: 0,
+      isClimateStrike: false,
+      showClimateStrikeBanner: false,
+      strikeDateString: `2019-11-29`
     };
   },
+  created() {
+    const climateStrikeDate = new Date(this.strikeDateString);
+    const today = new Date();
+
+    this.isClimateStrike = climateStrikeDate.getDate() === today.getDate() &&
+      climateStrikeDate.getMonth() === today.getMonth() &&
+      climateStrikeDate.getFullYear() === today.getFullYear();
+
+    this.showClimateStrikeBanner = climateStrikeDate.getDate() >= today.getDate() &&
+      climateStrikeDate.getMonth() >= today.getMonth() &&
+      climateStrikeDate.getFullYear() >= today.getFullYear();
```

Comment:
This would be false on 2019-10-31, as today's date (31) is greater than strike's month (29), but it would be true on 2019-10-01. So the banner would show up only then and now (if we released it earlier; it's november already so we could also don't care).

Just use this:
```
    this.showClimateStrikeBanner = today <= climateStrikeDate;
```

This will be false on strike day, but it won't be shown on strike day anyway.